src/app/component/header/header.component.scss

```diff
@@ -34,4 +34,9 @@ button:hover{
 }
 .modal .modal-header{
```

Comment:
here why whole modal class is used ? direct from boostrap ?

Reply:
I have Remove model